src/pywmapi/auction/api.py

```diff
@@ -0,0 +1,52 @@
+from typing import Dict, List, Optional, Tuple, Union
+from warnings import warn
+
+import requests
+
+from pywmapi.auction.model import AuctionItem, Auction
+
+__all__ = [
+    "create_auction",
+    # "get_item",
+]
+
+from pywmapi.auth import get_csrf_and_jwt
+from pywmapi.common import API_BASE_URL
+from pywmapi.exceptions import check_wm_response
+
+
```

Comment:
Hi, I found a trivial error here. The `lang` option is left blank, or the docstring is incorrect.

Also, I guess that this API func should take the user's login token, but here a new token is introduced? 
Maybe u could have a look on  `auth.models.Session` class and take it as a parameter. The user's info is included in the `Session` instance through the `signin` function.

BTW, I suppose that a unit test should be added under `test/` dir. :)